src/components/form/super_select/super_select_control.js

```diff
@@ -27,6 +28,7 @@ export const EuiSuperSelectControl = ({
       'euiSuperSelectControl--fullWidth': fullWidth,
       'euiSuperSelectControl--compressed': compressed,
       'euiSuperSelectControl-isLoading': isLoading,
+      'euiSuperSelectControl--isInvalid': isInvalid,
```

Comment:
Because this is a state modifier it should be a single dash
```suggestion
      'euiSuperSelectControl-isInvalid': isInvalid,
```